scripts/deploy-contracts.sh

```diff
@@ -39,7 +40,10 @@ fi
 # make deploy-base-anvil type=contract contract=DeployAccountFactory
 RIVER_BLOCK_TIME="${RIVER_BLOCK_TIME:-1}"
 
-cast rpc evm_setAutomine true --rpc-url $BASE_ANVIL_RPC_URL
+# Alternative nodes i.e. geth
+if [ "${BASE_EXECUTION_CLIENT}" != "geth_dev" ]; then
```

Comment:
What does `geth_dev` mean? I assume it means local development. But then it's sending rpc to anvil if not equal. shouldn't it be:
`"${BASE_EXECUTION_CLIENT}" == "geth_dev"` ?

Reply:
Yes, geth_dev means using a local geth node in dev mode, which does not support evm_setAutomine, so I want to skip it.
Alternatively, I could try/catch these cast commands

Reply:
I see. can you add this as a comment?